lib/logstash/util/compress.rb

```diff
@@ -0,0 +1,116 @@
+# encoding: utf-8
+require "logstash/namespace"
+require "logstash/environment"
+
+module LogStash::Util
+  module Zip
+
+    extend self
+
+    def extract(source, target)
```

Comment:
I think it's a good practice to document each method, description, parameters, return value, etc. for example see https://github.com/elastic/logstash/blob/master/lib/logstash/util/accessors.rb#L87-L92